code/__DEFINES/components.dm

```diff
@@ -148,6 +148,7 @@
 #define COMSIG_LIVING_EXTINGUISHED "living_extinguished"		//from base of mob/living/ExtinguishMob() (/mob/living)
 #define COMSIG_LIVING_ELECTROCUTE_ACT "living_electrocute_act"		//from base of mob/living/electrocute_act(): (shock_damage)
 #define COMSIG_LIVING_MINOR_SHOCK "living_minor_shock"			//sent by stuff like stunbatons and tasers: ()
+#define COMSIG_LIVING_CHARGE "living_charge"					//sent from borg recharge stations: (amount)

```

Comment:
maybe a better signal name could be used? living charge implies its only for living creatures but the correct way of saying it would be COMSIG_PROCESS_BORGCHARGER_OCCUPANT or something similar